internal/controller/dispatcher.go

```diff
@@ -154,19 +165,21 @@ func (r *Dispatcher) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 					return ctrl.Result{}, err
 				}
 
-				// TODO: Multicluster. This is where we create the placement that maps
-				// the appWrapper to the execution cluster. Only if the placement creation is successful
-				// will we do the update of the status to running/creating.
-				return ctrl.Result{}, fmt.Errorf("unimplemented multi-cluster synch path")
+				if err := r.createBindingPolicy(ctx, appWrapper); err != nil {
+					log.FromContext(ctx).Error(err, "Error while creating BindingPolicy object")
+					return ctrl.Result{}, err
+				}
 			}
 			return r.updateStatus(ctx, appWrapper, mcadv1beta1.Running, mcadv1beta1.Accepting)
 
 		case mcadv1beta1.Deleted:
 			if r.MultiClusterMode {
-				// TODO: Multicluster. This is where we delete the placement that maps
+				// TODO: Multicluster. This is where we delete the BindingPolicy that maps
```

Comment:
Update comment to remove TODO (or remove entirely if you think the code doesn't need a comment anymore)